azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -231,6 +198,7 @@ private HadoopSecurityManager_H_2_0(final Props props)
   private void disableFSCache() {
     this.conf.setBoolean(FS_HDFS_IMPL_DISABLE_CACHE, true);
     this.conf.setBoolean(FS_FAILOVER_IMPL_DISABLE_CACHE, true);
+    this.conf.setBoolean(FS_LOCAL_IMPL_DISABLE_CACHE, true);
```

Comment:
How is this change is tested?

Reply:
I reproduced the memory leak without this first fix first, verified through heap dump that there are leaked LocalFileSystem instances in File System cache.

Then deployed a version with this fix, verified through heap dump that these instances no longer exist.

Reply:
Thank you for thorough testing.